OMCompiler/Compiler/SimCode/SimCodeUtil.mo

```diff
@@ -279,7 +279,7 @@ protected
   HashTableSimCodeEqCache.HashTable eqCache;
   BackendDAE.Jacobian dataReconJac;
   BackendDAE.Variables setcVars,datareconinputvars;
-  list<SimCodeVar.SimVar> tmpsetcVars,tmpdatareconinputvars;
+  list<SimCodeVar.SimVar> tmpsetcVars,tmpdatareconinputvars,tmpaliasVars,tmpintAliasVars,tmpboolAliasVars,tmpstringAliasVars;
```

Comment:
I would prefer to have the declaration on separate lines (Just a matter of taste)